terraform/stacks/data_archive/variables.tf

```diff
@@ -2,3 +2,8 @@ variable "raw_data_bucket_name" {
   description = "The name of the bucket for raw sequencer output (BCLs)."
   default     = "org.umccr.data.raw-data-archive"
 }
+
+variable "oncoanalyser_bucket_name" {
+  description = "The name of the bucket for raw sequencer output (BCLs)."
```

Comment:
(don't hate me) Could you fix the description, pls? 😭

Reply:
Of course! ;-) 
While we are on it, I'll add some more changes to get a draft of the lifecycle rules. Stay tuned...